PORTAL/jobs/__init__.py

```diff
@@ -37,7 +36,7 @@ def __init__(self,
                  local_user: str,
                  worker: Optional[Union[str, _workers.WorkerConfig]],
                  data_dir: Optional[str] = None,
-                 **ignored
+                 **_
```

Comment:
I'm guessing this is to appease mypy.  Is there any way we can keep "ignored" (e.g. with some directive comment)?  "_" seems less clear.

Reply:
Even "_ignored" would be fine, if that's enough.

Reply:
Sorry -- it was just unintentional -- didn't mean to commit it.  This was a pyright suggestion I didn't mean to accept.  (Ultimately I think that's one to suppress -- parameter names for unused parameters are still pretty helpful as documentation).